packages/core/src/utils/pushNotification.ts

```diff
@@ -0,0 +1,86 @@
+import axios from 'axios';
+import config from '../config';
+import { client as prismic } from '../utils/prismic';
+import { NotificationType } from '../interfaces/app/appNotification';
+import { models } from '../database';
+import { AppUserModel } from '../database/models/app/appUser';
+import localesConfig from '../utils/locale.json';
+
+export async function sendNotification(
+    users: AppUserModel[],
+    type: NotificationType,
+) {
+    // registry notification
+    await models.appNotification.bulkCreate(users.map(el => ({
+        userId: el.id,
+        type,
+        isWallet: true,
+        isWebApp: true,
+    })));
+
+    // filter users that have walletPNT or appPNT
+    users = users.filter(el => el.walletPNT || el.appPNT);
```

Comment:
Actually, you have to filter by wallet. App push notifications are not setup, and they don't use FCM. I'm actually not sure if it even works.